features/step_definitions/api-token.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+
+/**
+ * Clean up: remove the test token
+ * @method
+ * @param  {String} tokenName
+ * @return {Promise}
+ */
+function cleanup(tokenName) {
+    return request({
+        uri: `${this.instance}/${this.namespace}/tokens`,
+        method: 'GET',
+        auth: {
+            bearer: this.jwt
+        }
+    }).then((response) => {
+        const match = JSON.parse(response.body)
+            .find(token => token.name === tokenName);
+
+        if (!match) return Promise.resolve();
+
+        return request({
+            uri: `${this.instance}/${this.namespace}/tokens/${match.id}`,
+            method: 'DELETE',
+            auth: {
+                bearer: this.jwt
+            }
+        }).then(deleteResponse => Assert.strictEqual(deleteResponse.statusCode, 204));
+    });
+}
+
+module.exports = function server() {
+    this.After(() => {
```

Comment:
Hooks typically go in the support dir.

https://github.com/screwdriver-cd/screwdriver/tree/master/features/support

https://github.com/cucumber/cucumber-js/blob/1.x/docs/support_files/hooks.md